index.html

```diff
@@ -429,5 +429,36 @@ <h2 class="about-title">
 				</li>
 			</ul>
 		</section>
+
+		<section class="contact">
+			<div class="contact-container">
+				<div class="contact-header">
+					<h2>Contact me</h2>
+					<p>
+						If you have an application you are interested in developing, a
+						feature that you need built or a project that needs coding. I’d love
+						to help with it
+					</p>
```

Comment:
- [ ] Please keep in mind that blank lines or indentations could lower the optimization of the website.
We only add them for good reasons such as the readability of the code when separating two logical pieces of code or two large sections apart.

> Please note that this is a mandatory requirement.
You can see it and similar important notes right [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

_Please apply it to similar situations in your code files._